modules/osqp-cpp/metadata.json

```diff
@@ -0,0 +1,17 @@
+{
+    "homepage": "https://github.com/google/osqp-cpp",
+    "maintainers": [
+        {
+            "email": "udayaprakash2899@gmail.com",
+            "github": "udaya2899",
+            "name": "Udaya Prakash Nageswaran"
```

Comment:
You might want to contact the owners of the upstream repo if they are willing to help with maintaining.